zne/noise_amplification/folding_amplifier/glorious_global_folding_amplifier.py

```diff
@@ -10,6 +10,8 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
+# pylint: disable=raise-missing-from
+
```

Comment:
Better to place this _inline_ where the error happens, if you write it here it will disable it in the entire file.